README.md

```diff
@@ -57,6 +57,7 @@ CREATE TABLE `customer_entity` (
 CREATE TABLE `sales_flat_order` (
   `entity_id` int UNSIGNED NOT NULL auto_increment COMMENT 'Entity Id' ,
   `grand_total` decimal(12,4) NULL COMMENT 'Grand Total' ,
+  `base_grand_total` decimal(12,4) NULL COMMENT 'Base Grand Total' ,
```

Comment:
There's another ticket to add the discount amount and it's not quite as simple. I can add a blank one though.